mlflow/store/db_migrations/versions/39d1c3be5f05_add_is_nan_constraint_for_latest_metrics_if_necessary.py

```diff
@@ -0,0 +1,33 @@
+"""add_is_nan_constraint_for_latest_metrics_if_necessary
```

Comment:
Nit: Can we rename this?

```suggestion
"""add_is_nan_constraint_for_metrics_tables_if_necessary
```

Reply:
Thanks for the catch!